ecleankernel/bootloader/grub.py

```diff
@@ -11,7 +11,11 @@
 class GRUB(LILO):
     name = 'grub'
     kernel_re = r'^\s*(kernel|module)\s*(\([^)]+\))?(?P<path>\S+)'
-    def_path = ('/boot/grub/menu.lst', '/boot/grub/grub.conf')
+    def_path = (
+                os.getenv("GRUB_CFG"),
```

Comment:
```suggestion
    def_path = (os.getenv("GRUB_CFG"),
```

i.e. either use hanging indent, or one "tab" in the next lines.